### ISSUE TITLE
error installing 

### ISSUE THREAD

**Issue by amit777** [ryanwinchester/netsuite-php#21]
when running this:
```
composer require "ryanwinchester/netsuite-php:2016.2.*"
```

I get this error:
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package ryanwinchester/netsuite-php 2016.2.* exists as ryanwinchester/netsuite-php[dev-develop, dev-master, v1.1.0, v2.0.0-alpha, v2.0.1-alpha, v2.0.2-alpha, v2.0.3-alpha, v2.0.4-alpha, v2014.2.0, v2015.1.0, v2015.1.1, v2015.1.2, v2015.2.0, v2015.2.1, v2016.1.0, v2016.1.1, v2016.1.2] but these are rejected by your constraint.
```

However It works when I run with slightly older version
```
composer require "ryanwinchester/netsuite-php:2016.1.*"
```

**Comment by ryanwinchester** [comment]
Sorry, I merged the 2016.2 last night but did not tag it yet

**Comment by ryanwinchester** [comment]
should be good now
